app/Main.hs

```diff
@@ -28,10 +28,11 @@ import DynamicGraphs.WriteRunDot (generatePrereqsForCourses)
 taskMap :: Map.Map String ([String] -> IO ())
 taskMap = Map.fromList [
     ("server", const runServer),
-    ("database", const setupDatabase),
+    ("database", const runDatabase),
```

Comment:
Overall this is a good time to fix up some of the naming of this command and the `graphs` command. Change the following:

- the command `database` to `database-courses`
- the command `graphs` to `database-graphs`
- the function `runDatabase` to `populateCourseInfo`

and make sure to update the README instructions as well!